docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -169,6 +218,83 @@ SLIs for Monitor-based SLOs are calculated at a granularity of 1 minute. A minut
+
+```sql
+_view=sumologic_slo_output sloId="<your-SLO-ID>"
+| where [subquery: _view=sumologic_slo_output sloId="<your-SLO-ID>"
+| max(sloVersion) as sloVersion | compose sloVersion]
+-- (replace with a valid SLO Id)
+```
+
+These log messages will be delayed by one hour, as the system ensures consistency to account for ingest delay of source telemetry.
+
+
+### SLO Lookup Tables
+
+You can call a SLO Lookup Table to view all SLO metadata in your environment. These tables reside under a fixed path, `sumo://content/slos`. Data is managed and refreshed automatically on our end.
+
+There are two ways to use it:
+
+* To join the results of your SLO precomputed data from `_view=sumologic_slo_output` with your metadata contained in the internal lookup table based on the joining key (`sloId`, `sloVersion`):
+  ```sql
+  _view=sumologic_slo_output
+  | lookup * from sumo://content/slos on sloId, sloVersion
+  ```
+* To enlist the contents of the lookup table:
+  ```sql
+  cat sumo://content/slos
+  ```
+
+#### Example 1
+
+As an example, say you had a SLO [dashboard](/docs/dashboards-new) and wanted to see error budget burndown from several of your apps and services combined.<br/><img src={useBaseUrl('img/observability/percent-error-remain.png')} alt="percent-error-remain" width="450"/>
+
+You would need to create a custom graphic that combines multiple SLOs from multiple services:
+
+1. Go to **Manage Data** > **Monitoring** > **SLO**.
+1. Click on any SLO line item.
+1. Hover over the **Percentage budget remaining** panel, then click the three-dot icon > **Open in Log Search**.<br/><img src={useBaseUrl('img/observability/open-in-logsearch.png')} alt="open-in-logsearch" width="150"/>
+1. In the search field, enter the following snippet. This will join data from multiple sources for your lookup table.
+  ```sql
+  _view=sumologic_slo_output
+  | lookup * from sumo://content/slos on sloId, sloVersion
+  | where !isBlank (sloname) and slofolderpath matches "*"
+  | concat (sloname, " (", sloId, ")") as sloUniqueName
+  | sum (goodCount) as goodEvents, sum(totalCount) as totalEvents, last (compliancetarget) as target, last(slofolderpath) as sloPath, last(sliwindowsize) as sliwindowsize, last(slievaluationtype) as evaluationType by sloUniqueName
+  | totalEvents - goodEvents as badEvents
+  | if (evaluationType = "Window", queryTimeRange() / 1000 / sliwindowsize, totalEvents) as denominator
+  | 100 * (1 - badEvents / denominator) as sli
+  | 100 * (sli - target) / (100 - target) as budgetRemaining
+  | fields sloUniqueName, budgetRemaining
+  ```
+1. Click **Add to Dashboard**.<br/><img src={useBaseUrl('img/observability/add-to-dashboard.png')} alt="add-to-dashboard" width="200"/>
+
+
+#### Example 2
+
+You can leverage SLO tags in **Log Search** queries. Here's a query, for example, to display all SLOs that have tags:
+
+<img src={useBaseUrl('img/observability/slo-tags-query-log.png')} alt="slo-tags-query.png" /><br/>
```

Comment:
Hi @tarunk2 , about [this comment](https://github.com/SumoLogic/sumologic-documentation/pull/2275/files#r1214309385), I did have include a bit about tags in SLO lookup tables further down here in the doc. I'll combine the example you shared with mine and then cross-link those sections.